authorship/data.py

```diff
@@ -101,34 +93,41 @@ def read_session(lines):
     derive the set of the most frequent words
     """
     lines_to_words = lines.read()
-    lines_to_words = re.sub('[0-9]*\.[0-9]*\t', '', lines_to_words)
+    lines_to_words = re.sub('[0-9]*\.[0-9]*\t', '', lines_to_words)  # to remove timestamps
```

Comment:
This seems to be specific to a particular file format. Maybe for future reference, make this explicit, also in the name of the class and/or module.

Reply:
class renamed